src/types/Scene.tsx

```diff
@@ -38,6 +38,10 @@ export interface Project {
   latitude: number;
   longitude: number;
   orientation: number;
+  createdAt: number;
+  modifiedAt: number;
+  thumb: string;
+  id?: string;
```

Comment:
why '?'?

Reply:
`id` is related to database only and cannot be declared when creating the project to be added to the database